frontend/src/api/openapi/services/AnalysisService.ts

```diff
@@ -122,7 +122,7 @@ export class AnalysisService {
       url: "/analysis/timeline_analysis",
       query: {
         project_id: projectId,
-        threshold: threshold / 100.0,
```

Comment:
Not sure where this change is coming from, @bigabig any idea?

Reply:
I think we'll need to update the frontend code to perform the division by 100 elsewhere. The frontend is sending values between 0 and 100 right now.